operator/api/server.go

```diff
@@ -137,9 +138,7 @@ func (s *server) Start(ctx cell.HookContext) error {
 		}
 		s.httpSrvs[i].listener = ln
 
-		s.logger.WithFields(logrus.Fields{
-			fmt.Sprintf("address-%d", i): ln.Addr().String(),
-		})
+		s.logger.With(fmt.Sprintf("address-%d", i), ln.Addr().String())
```

Comment:
Did this ever do anything?  the `With` family of methods returns a new logger with added context, which is discarded in this case (both in new and old code)